_content/articles/lasting-inspiration_tathaloka.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Lasting Inspiration"
+authors:
+  - tathaloka
+external_url: "https://www.bhikkhuni.net/wp-content/uploads/2013/06/Lasting-Inspiration.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1PhwcQTYb6yezrGakveBjSWcCWN0JAcVl/view?usp=sharing"
+course: avadana
+tags:
+  - rebirth-stories
+  - karma
+  - bodhicitta
```

Comment:
So far I've been tagging all the Bodhicitta related items with either `bodhisattva` or (right) `thought` depending on whether it's specifically about the Bodhisattva vow or about the desire for awakening in general.